apps/dashr-brain-viewer/app.R

```diff
@@ -0,0 +1,455 @@
+
+library(dashColorscales)
+library(dashR)
+library(plotly)
+library(rapportools)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+library(devtools)
+library(rjson)
+
+appName <- Sys.getenv("DASH_APP_NAME")
+
+appName <- Sys.getenv("DASH_APP_NAME")
+if (appName != ""){
+  pathPrefix <- sprintf("/%s/", appName)
+  
+  Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
+             DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+  
+  setwd(sprintf("/app/apps/%s", appName))
+}
+
+DEFAULT_COLORSCALE <- list(list(0,'#0c3383'), list(0.25,'#0a88ba'), list(0.5,'#f2d338'),
+                           list(0.75,'#f28f38'), list(1,'#d91e1e'))
+
+DEFAULT_COLORSCALE_NO_INDEX <- list()
+
+for (i in 1:length(DEFAULT_COLORSCALE)) {
```

Comment:
I noticed that only DEFAULT_COLORSCALE_NO_INDEX is used in the rest of the code, so maybe instead of using a for loop to fill DEFAULT_COLORSCALE_NO_INDEX from DEFAULT_COLORSCALE, you could just initialize the list for DEFAULT_COLORSCALE_NO_INDEX right away.